### DISCUSSION TITLE
Is it possible to hide a tile?

### DISCUSSION THREAD

**Discussion by ssandino** [firecmsco/firecms#362] [General]
We tried, but haven't been lucky? Maybe just not possible or did we oversee something?

(the option to not map it entirely is not an option, as we use it as subcollection)

<img width="1047" alt="Screen Shot 2022-08-13 at 15 28 52" src="https://user-images.githubusercontent.com/6095849/184496300-55365128-d48e-4556-8d4f-cdcfcecb0c72.png">

**Comment by fgatti675** [comment]
Hi @ssandino 
When you say you use it as a subcollection, you mean you have some reference that points to it?

**Reply by ssandino** [reply]
Yes!

**Reply by ssandino** [reply]
Feel free to delete the question - it would need more details to make sense ;-)

**Reply by fgatti675** [reply]
You can now hide a collection from the main navigation with `hideFromNavigation` :)
